ee/codegen/src/generators/chat-message-content.ts

```diff
@@ -37,7 +38,7 @@ class StringChatMessageContent extends AstNode {
       arguments_: [
         python.methodArgument({
           name: "value",
-          value: python.TypeInstantiation.str(value),
+          value: python.TypeInstantiation.str(removeEscapeCharacters(value)),
```

Comment:
[oos] I get the sense we're going to see something like this in all usages of `python.TypeInstantiation.str`. It may be worth opening a PR to that repo with a test that reproduces the bug on their end and implementing a fix that escapes properly rather than removes it. I have a creeping concern that removal could lead to data loss, which was fine for input names but seems less tolerant for input values